analyses/malawi/flooding_trigger/03_combine_events.md

```diff
@@ -0,0 +1,173 @@
+### Combining historical flood events
+
+This notebook combines various sources of historical flood event data: 1) Floodscan derived events, 2) events reported by the RCO, 3) DoDMA reported flooding years, and 4) past CERF allocations.
+
+```python
+from datetime import datetime
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import os
+from pathlib import Path
+import sys
+from datetime import timedelta
+
+mpl.rcParams['figure.dpi'] = 300
+
+path_mod = f"{Path(os.path.dirname(os.path.abspath(''))).parents[1]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding.config import Config
+
+config = Config()
+
+SAVE_PLOT = False
+SAVE_DATA = False
+
+PLOT_DIR = config.DATA_DIR / 'processed' / 'mwi' / 'plots' / 'flooding'
+PRIVATE_DIR = config.DATA_PRIVATE_DIR
+EXPLORE_DIR = PRIVATE_DIR / 'exploration' / 'mwi' / 'flooding'
+
+stations_adm2 = {
+    'G1724': 'Nsanje',
+    'G2001': 'Chikwawa'
+}
+```
+
+Read in the various event datasets. Both the RCO and Floodscan events are in the same format with defined start and end dates and whereby each event corresponds to a district.
+
+```python
+df_emdat = pd.read_csv(EXPLORE_DIR / 'emdat.csv')
+df_dodma = pd.read_csv(PRIVATE_DIR / 'processed' / 'mwi' / 'mvac_dodma_flood_district.csv')
```

Comment:
Possibly also add that the dodma was shared privately? Just so it is clear what the sources of the data files are